pom.xml

```diff
@@ -5,7 +5,7 @@
     <groupId>org.mvel</groupId>
     <artifactId>mvel2</artifactId>
     <packaging>jar</packaging>
-    <version>2.4.14.Final</version>
+    <version>2.4.14.BasedirFix</version>
```

Comment:
You should have started with the master branch rather than the current release branch. Also this line should only be updated when publishing a release. I think this creates a merge conflict since the value is currently 2.4.15-SNAPSHOT in the master branch.